.github/docker/centreon-poller/alma9/entrypoint/container.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+#set -e
+set -x
+
+# Run each startup script located in BASEDIR.
+# ls is required to ensure that the scripts are properly sorted by name.
+BASEDIR="/usr/share/centreon/container.d"
+for file in `ls $BASEDIR` ; do
```

Shellcheck (0.10.0)

[error] Iterating over ls output is fragile. Use globs.

(SC2045)